substra/cli/interface.py

```diff
@@ -112,6 +112,15 @@ def click_global_conf_config(f):
         help='Config path (default ~/.substra).')(f)
 
 
+def click_global_conf_metadata(f):
+    """Add metadata option to command."""
+    return click.option(
+        '--metadata',
+        callback=update_global_conf,
```

Comment:
Metadata is an option that is only available for `add` commands, and it is forwarded to the API. The global conf option on the other hand is here to tell the cli how to behave locally (output format, which conf to use etc.) I don't think they should be mixed together.